### PR TITLE
Add Iso between Option and Disjunction

### PR DESCRIPTION
**Pull request by aoiroaoino**
#220

### CODE REVIEW

File: core/src/main/scala/monocle/std/Option.scala [hunk 1]
```diff
@@ -15,6 +15,11 @@ trait OptionFunctions {
   def none[A]: Prism[Option[A], Unit] =
     Prism[Option[A], Unit]{ case None => Some(()); case Some(_) => None }(_ => None)
 
+  def pOptionToDisjunction[A, B]: PIso[Option[A], Option[B], Unit \/ A, Unit \/ B] =
+    PIso[Option[A], Option[B], Unit \/ A, Unit \/ B](_.map(\/-(_)) getOrElse -\/(()))(_.toOption)
+
+  def optionToDisjunction[A]: Iso[Option[A], Unit \/ A] =
+    Iso[Option[A], Unit \/ A](_.map(\/-(_)) getOrElse -\/(()))(_.toOption)
```
Comment (julien-truffaut) [inline core/src/main/scala/monocle/std/Option.scala]: if you want, you can simply define `optionToDisjunction ` in terms of `pOptionToDisjunction `

Comment (aoiroaoino) [inline core/src/main/scala/monocle/std/Option.scala]: Oops, I'll fix it

**Comment by julien-truffaut** [comment]
Thanks!

**Comment by aoiroaoino** [comment]
My pleasure :smile:

**Pull request merged**